main.go

```diff
@@ -356,6 +356,7 @@ func (p Proxy) getTokenFromAPI(ipAddress string) string {
 		return ""
 	}
 
+	defer resp.Body.Close()
 	token, err := io.ReadAll(resp.Body)
```

Comment:
What are your thoughts about using resp.Body.Read() instead of io package directly?

Reply:
It's more complex. What would be the advantage?

Reply:
I don't give any thought to standard library imports. Should I?